experiments/20160805-mutagenesis/20160831-mutagenesis.md

```diff
@@ -100,7 +100,7 @@ Only PCR with P1 (band at 2.7kb) appears successful.
 
 | Fragment        | expected size |
 |---------------|---------------|
-| 1: PB2 Part 1  | 1600          |
-| 2: PB2 Part 2 | 820           |
-| 3: pCI Part 1  | 2707          |
-| 4: pCI Part 2  | 1374          |
+| A1  | 1600          |
```

Comment:
Sorry! I should have been clearer - it's most clear when A1 and the fragment name "PB2 Part 1" are put together.